content/site.json

```
@@ -14,7 +14,7 @@
       "link": "/profile"
     }
   ],
-  "announcement": "Polygon hardfork is happening 17 Jan 2023. There may be disruptions in the publishing and consuming of assets.",
+  "announcement": "Lock your OCEAN to get veOCEAN, earn rewards and curate data.",
```

think there was a link also here
```suggestion
  "announcement": "[Lock your OCEAN](https://df.oceandao.org/) to get veOCEAN, earn rewards and curate data.",
```